### PR TITLE
docs(rpc-go): refining rpc-go commands

### PR DESCRIPTION
**Pull request by Kimberly-Davis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

        modified:   docs/GetStarted/buildRPC.md

## Anything the reviewer should know when reviewing this PR?

RB ran into a few gotchas, which this check-in should fix.

### CODE REVIEW

File: docs/GetStarted/buildRPC.md [hunk 1]
```diff
@@ -58,6 +70,9 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
 2. Confirm the version numbers.
 
 ## Get the RPC
+If you have already cloned the toolkit repository as described in [Set Up](setup.md), change to the cloned `rpc-go` directory and see [Build the RPC](#buildtherpc).
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:73]: You don't actually need to reference an anchor, the markdown headings have this built in, you can reference (#build-the-rpc)

File: docs/GetStarted/buildRPC.md [hunk 2]
```diff
@@ -71,10 +86,12 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
    cd rpc-go
    ```
 
-## Build the RPC
+##  <a name="buildtherpc"></a>Build the RPC
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:89]: Remove anchor

File: docs/GetStarted/buildRPC.md [hunk 3]
```diff
@@ -107,16 +124,17 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
...
+3. Confirm a successful build:
 
     === "Linux"
         ``` bash
         sudo ./rpc version
         ```
     === "Windows"
         ``` bash
-        .\rpc version
+        rpc version
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:137]: We need the '.\\' on all Windows cmds, while Command Prompt doesn't care one way or the other, it is for Powershell.  This allows the copy/paste to work across both.

File: docs/GetStarted/buildRPC.md [hunk 4]
```diff
@@ -107,16 +124,17 @@ Developed in Go* programming language, the Remote Provisioning Client (RPC) appl
           129 |  ILibSpawnNormalThread((voidfp)(&LME_Thread), module);
           ...
         ```
+2. Copy to the executable to a managed device.
```
Comment (bwendlandt-intel) [inline docs/GetStarted/buildRPC.md:127]: Grammar: 'Copy the executable...'

**Review by bwendlandt-intel — CHANGES_REQUESTED** [review]

**Review by rjbrache — APPROVED** [review]
These mods add clarity.  Fix BW's findings and things look good from a content and flow perspective.

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**